docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -149,15 +149,16 @@ Both offer monitoring of a single process instance, its variables and path, inci
 
 ## Limitations and availability
 
-Play is being rebuilt and progressively rolled out to more users. This section explains why you might not see the **Play** tab, and any additional limitations.
```

Comment:
I think we can remove this now, wdyt? @HanselIdes

Reply:
Yep, good idea!